Updating pre-flight doc to include checking spindle rotation (#95)
MillenniumMachines/millenniummachines.github.io: docs/milo/manual/chapters/110_pre_flight_checks.md

---

## Verifying Spindle Rotation

Before you can start cutting material, you need to make sure that the spindle is wired correctly and is rotating in the proper direction.

> Using a 6mm gauge pin (there should be one included in the LDO kit) draw adraw a stright line ~10mm down the side of the gauge pin and insert it into a 6mm collet in your spindle. 

Reviewer: ```suggestion
Using a 6mm gauge pin (included in most LDO kits) draw a stright line ~10mm down the side of the gauge pin and insert it into a 6mm collet in your spindle. 
```

!!! note

>     Make sure to snugly tighten the gauge pin in the spindle before processing to the next step

Reviewer: ```suggestion
    Make sure to snugly tighten the gauge pin in the spindle before proceeding to the next step!
```

> Set your spindle to 10k RPM and then turn it off and watch the rotation of the line as the spindle slows down.  

Reviewer: ```suggestion
Start your spindle at the lowest speed and then turn it off and watch the rotation of the line on the gauge pin as the spindle slows down.  
```

- If the line on the gauge pin is going from right to left ( <-- ), your spindle is rotating in the right direction.  

> - If the line is going from left to right ( --> ), the spindle is rotating in the wrong direction and you will need to flip two wires in the UVW set coming from the spindle.

Reviewer: ```suggestion
- If the line is going from left to right ( --> ), the spindle is rotating in the wrong direction. Swap any two spindle phase wires (UVW) at the VFD terminals (Swap UV, UW or VW).
```
